webapp/packages/plugin-data-viewer/src/TableViewer/ValuePanel/Presentation/BooleanValue/BooleanValuePresentation.tsx

```diff
@@ -9,49 +9,35 @@ import { observer } from 'mobx-react-lite';
 
 import { Radio, TextPlaceholder, useTranslate } from '@cloudbeaver/core-blocks';
 import type { TabContainerPanelComponent } from '@cloudbeaver/core-ui';
+import { isDefined } from '@cloudbeaver/core-utils';
 
-import { ResultSetEditAction } from '../../DatabaseDataModel/Actions/ResultSet/ResultSetEditAction';
-import { ResultSetFormatAction } from '../../DatabaseDataModel/Actions/ResultSet/ResultSetFormatAction';
-import { ResultSetSelectAction } from '../../DatabaseDataModel/Actions/ResultSet/ResultSetSelectAction';
-import { ResultSetViewAction } from '../../DatabaseDataModel/Actions/ResultSet/ResultSetViewAction';
-import type { IDatabaseResultSet } from '../../DatabaseDataModel/IDatabaseResultSet';
-import type { IDataValuePanelProps } from '../../TableViewer/ValuePanel/DataValuePanelService';
+import { getResultSetActions } from '../../../../DatabaseDataModel/Actions/ResultSet/getResultSetActions';
+import type { IDatabaseResultSet } from '../../../../DatabaseDataModel/IDatabaseResultSet';
+import type { IDataValuePanelProps } from '../../DataValuePanelService';
 import classes from './BooleanValuePresentation.module.css';
-import { isStringifiedBoolean } from './isBooleanValuePresentationAvailable';
+import { useValuePanelBooleanValue } from './useValuePanelBooleanValue';
 
 export const BooleanValuePresentation: TabContainerPanelComponent<IDataValuePanelProps<any, IDatabaseResultSet>> = observer(
   function BooleanValuePresentation({ model, resultIndex }) {
     const translate = useTranslate();
-    const selection = model.source.getAction(resultIndex, ResultSetSelectAction);
-    const activeElements = selection.getActiveElements();
 
-    if (activeElements.length === 0) {
-      return null;
-    }
-
-    let value: boolean | null | undefined;
-
-    const view = model.source.getAction(resultIndex, ResultSetViewAction);
-    const editor = model.source.getAction(resultIndex, ResultSetEditAction);
+    const { selectAction, viewAction, editAction, formatAction } = getResultSetActions({ model, resultIndex });
```

Comment:
I wouldn't say I like this way of usage; now, this method will initialize unused actions, and it's easy to expand this function in the future to get performance issues or unwanted behavior. Database actions should be lazy and initialized only when requested.